### ISSUE TITLE
KML Polylines have ArcType value as either undefined or ArcType.NONE after import

### ISSUE THREAD

**Issue by JoshuaKahn** [CesiumGS/cesium#9555]
Sandcastle example: None Specifically

Browser: Chrome

Operating System: Linux (Kubuntu) 20.04

When importing a KML into Cesium, polylines will either have their ArcType value set as `undefined` or `ArcType.NONE`. 

[Here's a polyline printed to the log after import.](https://aws1.discourse-cdn.com/business4/uploads/cesium/original/2X/1/182835c9a79ef6c8dd76a4b858ee79d27be8f303.png)

I discovered this when trying to force polylines from Cesium to be clamped to terrain upon import. Often without the right `ArcType` value the engine will crash trying to get the line to clamp, with the error:

```CesiumWidget.js?d9c1:690 An error occurred while rendering.  Rendering has stopped.
DeveloperError: Valid options for arcType are ArcType.GEODESIC and ArcType.RHUMB.```

I think whether or not the line has `tessellate` in the KML file determines what ArcType is chosen. If `tessellate` is present, then ArcType will be `undefined`; if it's not, then `ArcType` will be `ArcType.NONE`.


An example KML file [can be located here.](https://community.cesium.com/uploads/short-url/v0vvZBzzE6L5vBOW7S2iBjqm2e1.kml)

**Comment by ebogo1** [comment]
@JoshuaKahn - are you forcing the polyline to clamp with the `clampToGround` option? Currently we ignore this option if the `tesselate` flag is missing. 

> I think whether or not the line has tessellate in the KML file determines what ArcType is chosen.

This seems accurate - see these lines in [KmlDataSource.js](https://github.com/CesiumGS/cesium/blob/a02af5ed809b09d3b84381597cccf94ab8b9007a/Source/DataSources/KmlDataSource.js#L1681-L1683): 
```
if (!tessellate || canExtrude) {
    polyline.arcType = ArcType.NONE;
}
```

@lilleyse Do you know what the correct behavior should be between `ArcType` and `tesselate`?

**Comment by JoshuaKahn** [comment]
> @JoshuaKahn - are you forcing the polyline to clamp with the `clampToGround` option? Currently we ignore this option if the `tesselate` flag is missing.
> 
> > I think whether or not the line has tessellate in the KML file determines what ArcType is chosen.
> 
> This seems accurate - see these lines in [KmlDataSource.js](https://github.com/CesiumGS/cesium/blob/a02af5ed809b09d3b84381597cccf94ab8b9007a/Source/DataSources/KmlDataSource.js#L1681-L1683):
> 
> ```
> if (!tessellate || canExtrude) {
>     polyline.arcType = ArcType.NONE;
> }
> ```
> 
> @lilleyse Do you know what the correct behavior should be between `ArcType` and `tesselate`?

@ebogo1  Yes I am forcing the polyline to clamp as I've run into circumstances in the pass where KMLs I have been sent do not have tessellate enabled, meaning them easily clip into terrain. The tool I'm developing Cesium with will be used by other users, and I'm hoping that I can enable KML import into the application such that they don't have to edit any KML file they are given to have polylines clamp to terrain. 

Right now I'm avoiding the resulting error mentioned above by setting polylines to use `ArcType.GEODESIC`, then setting their clamp to ground value to `true`.

**Comment by ebogo1** [comment]
@JoshuaKahn For now your approach might be best. We'll leave this issue open to revisit in the future - the end goal might be to add an option to `KmlDataSource` that overrides `clampToGround` behavior, but I'm unsure when we'll get to resolving this.
